frontend/javascripts/oxalis/view/right-border-tabs/dataset_info_tab_view.tsx

```diff
@@ -525,24 +526,22 @@ class DatasetInfoTabView extends React.PureComponent<Props, State> {
     }
   }
 
-  maybePrintOwner() {
+  maybePrintOwnerAndContributors() {
     const { activeUser } = this.props;
-    const { owner } = this.props.tracing;
+    const { owner, contributors, annotationId, annotationType, othersMayEdit } = this.props.tracing;
 
     if (!owner) {
       return null;
     }
 
-    if (!activeUser || owner.id !== activeUser.id) {
-      return (
-        <span>
-          Owner: {owner.firstName} {owner.lastName}
-        </span>
-      );
-    }
-
-    // Active user is owner
-    return null;
+    return (
+      <div>
+        <div>Owner: {formatUserName(activeUser, owner)}</div>
+        <div>
+          Contributors: {contributors.map((user) => formatUserName(activeUser, user)).join(", ")}
```

Comment:
when there are no contributors, this should read a bit nicer than `contributors:`.